code/modules/hallucination/on_fire.dm

```diff
@@ -37,7 +37,6 @@
 	return ..()
 
 /datum/hallucination/fire/start()
-	hallucinator.set_fire_stacks(max(hallucinator.fire_stacks, 0.1)) //Placebo flammability
```

Comment:
What does this do?

Reply:
Nothing much apparently, but it deletes out the wet stacks for no reason, which is bonkers for an hallucination.